apis/apiextensions/v1alpha1/crds.go

```diff
@@ -0,0 +1,287 @@
+        type: string
+      fieldPath:
+        description: 'If referring to a piece of an object instead of an
+          entire object, this string should contain a valid JSON/Go field
+          access statement, such as desiredState.manifest.containers[2].
+          For example, if the object reference is to a container within
+          a pod, this would take on a value like: "spec.containers{name}"
+          (where "name" refers to the name of the container that triggered
+          the event) or if no container name is specified "spec.containers[2]"
+          (container with index 2 in this pod). This syntax is chosen only
+          to have some well-defined way of referencing a part of an object.
+          TODO: this design is not final and this field is subject to change
+          in the future.'
+        type: string
+      kind:
+        description: 'Kind of the referent. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+        type: string
+      name:
+        description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names'
+        type: string
+      namespace:
+        description: 'Namespace of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/'
+        type: string
+      resourceVersion:
+        description: 'Specific resourceVersion to which this reference is
+          made, if any. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#concurrency-control-and-consistency'
+        type: string
+      uid:
+        description: 'UID of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#uids'
+        type: string
+    required:
+    - apiVersion
+    - kind
+    - name
+  type: array
+writeConnectionSecretToRef:
+  description: WriteConnectionSecretToReference specifies the name of
+    a Secret, in the same namespace as this resource claim, to which any
+    connection details for this resource claim should be written. Connection
+    details frequently include the endpoint, username, and password required
+    to connect to the managed resource bound to this resource claim.
+  properties:
+    name:
+      description: Name of the secret.
+      type: string
+  required:
+  - name
+  type: object
+`
+
+// InfraStatusProps is OpenAPIV3Schema for status fields that Crossplane uses for
+// Infrastructure kinds.
+const InfraStatusProps = `
+bindingPhase:
+  description: Phase represents the binding phase of a managed resource
+    or claim. Unbindable resources cannot be bound, typically because
+    they are currently unavailable, or still being created. Unbound resource
+    are available for binding, and Bound resources have successfully bound
+    to another resource.
+  enum:
+  - Unbindable
+  - Unbound
+  - Bound
+  - Released
+  type: string
```

Comment:
Yeah, I'll remove binding phase. But it's worth to talk about its implications. There are two things binding phase does:
* The relation representation.
  * Per the design spec, this is handled via other means and I just left a [comment](https://github.com/crossplane/crossplane/pull/1163#discussion_r410883552) on that section.
* The readiness signal for propagation.

I think the first one is solved via other means as you pointed out but the second one is not and IMO it's necessary because the propagation starts only if the underlying resource is bound, i.e. ready, to the namespaced claim. This is a good feature since pods mounting the secret will not be started until the secret is created. However, binding phase is an overkill for this. Somehow, the namespaced publication instance should know that the cluster-scoped instance is ready. Since the CRD is defined by us, `Ready` condition would serve this purpose. @negz does that make sense to you? If so, it's worth making an addition to one-pager.